apps/meeting/src/app.tsx

```diff
@@ -28,23 +29,25 @@ const App: FC = () => (
         <NotificationProvider>
           <Notifications />
           <ErrorProvider>
-            <MeetingProvider {...meetingConfig}>
-              <NavigationProvider>
-                <Switch>
-                  <Route exact path={routes.HOME} component={Home} />
-                  <Route path={routes.DEVICE}>
-                    <NoMeetingRedirect>
-                      <DeviceSetup />
-                    </NoMeetingRedirect>
-                  </Route>
-                  <Route path={routes.MEETING}>
-                    <NoMeetingRedirect>
-                      <MeetingModeSelector />
-                    </NoMeetingRedirect>
-                  </Route>
-                </Switch>
-              </NavigationProvider>
-            </MeetingProvider>
+            <VoiceFocusProvider>
```

Comment:
just one question - does this need to be a parent of MeetingProvider?

Reply:
Nope, the builder can put the VF provider wherever they want in the component tree. And Meeting provider and VF provider are independent from each other. VF provider will initial the VF workflow itself.